### ISSUE TITLE
OpenPose for TX2 JetPack 4.2 Installation Scripts

### ISSUE THREAD

**Issue by mdmorar** [CMU-Perceptual-Computing-Lab/openpose#1390]
# Issue Summary
I wanted to know if you guys plan to add support for the TX2 JetPack 4.2 as its running Ubuntu 18.04 and the older shell scripts for JetPack 3.3 do not work anymore on this new platform. I was able to build them and install all the caffe elements with the new OpenCV 3.4.6, but it immediately shuts down my TX2 after calling openpose.bin or even the 01 example that is provided. 

The assumption is that its not built for that architecture and the caffe as well as open pose needs to be built against the 6.2 aarch, but again I'm not sure I'm fighting this as we speak. 

### Executed Command (if any)
nvidia@nvidia-desktop:~/datacapture/openpose$ ./build/examples/tutorial_api_cpp/01_body_from_image_default.bin 

### OpenPose Output (if any)
Standard cout commands but when trying to run it shuts down my device directly, so no output available in terms of errors in terminal.

### Errors (if any)
Shuts down device after trying to start openpose. I get all the cout's from the c++ files but after the last one in example one it dies. 

### Type of Issue
- Execution error
Shuts down the entire TX2 device after running open pose against the example or webcam.
- Help wanted
I am working on modifying the current shell scripts locally to try and figure out if its a caffe aarch issue or openpose aarch issue or both?
- Enhancement / offering possible extensions / pull request / etc
I was wondering if you guys can get some new shell scripts for Caffe and JetPack4.2 as you created some very good ones for JetPack3.3 9 months ago and I was able to get it working on JetPack3.3 
### Your System Configuration
1. **Whole console output** (if errors appeared), paste the error to [PasteBin](https://pastebin.com/) and then paste the link here: 
None, shuts down my device.
2. **OpenPose version**: Latest GitHub code? Or specific commit (e.g., d52878f)? Or specific version from `Release` section (e.g., 1.2.0)?
Latest github code pulled.
3. **General configuration**:
    - **Installation mode**: CMake, sh script, manual Makefile installation, ... (Ubuntu); CMake, ... (Windows); ...?
cmake-gui-3.15.1
cmake version 3.15.1
    - **Operating system** (`lsb_release -a` in Ubuntu): 
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic
    - **Operating system version** (e.g., Ubuntu 16, Windows 10, ...):
Ubuntu 18.04 L4T, Nvidia JetPack 4.2 
    - **Release or Debug mode**? (by default: release):
Release
    - Compiler (`gcc --version` in Ubuntu or VS version in Windows): 5.4.0, ... (Ubuntu); VS2015 Enterprise Update 3, VS2017 community, ... (Windows); ...?
g++ (Ubuntu/Linaro 7.4.0-1ubuntu1~18.04.1) 7.4.0
4. **Non-default settings**:
    - **3-D Reconstruction module added**? (by default: no):
    - Any other custom CMake configuration with respect to the default version? (by default: no):

5. **3rd-party software**:
    - **Caffe version**: Default from OpenPose, custom version, ...?
Default
    - **CMake version** (`cmake --version` in Ubuntu):
CMake 3.15.1
    - **OpenCV version**: pre-compiled `apt-get install libopencv-dev` (only Ubuntu); OpenPose default (only Windows); compiled from source? If so, 2.4.9, 2.4.12, 3.1, 3.2?; ...?
OpenCV 3.4.6

6. If **GPU mode** issue:
    - **CUDA version** (`cat /usr/local/cuda/version.txt` in most cases):
10.0, V10.0.166
    - **cuDNN version**:
7.3.1.28-1+cuda10.0 arm64
    - **GPU model** (`nvidia-smi` in Ubuntu):
TX2 OnBoard GPU

7. If **CPU-only mode** issue:
    - **CPU brand & model**:
    - Total **RAM memory** available:
8. If **Python** API:
    - **Python version**: 2.7, 3.7, ...?
Both
    - **Numpy version** (`python -c "import numpy; print numpy.version.version"` in Ubuntu):
1.16.5
9. If **Windows** system:
    - Portable demo or compiled library?
N/A

10. If **speed performance** issue:
    - Report OpenPose timing speed based on [this link](https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/speed_up_openpose.md#profiling-speed).
N/A

**Comment by gineshidalgo99** [comment]
Try not using the shell scripts, but rather the CMake steps (normal Ubuntu installation). They should work. Please, let us know if that works, or otherwise which changes you had to do, as we can't test on that hardware. Thanks!

**Comment by mdmorar** [comment]
Hi,

Well I tried using the normal installation steps at the very beginning and since this did not work I had to try the JetPack 3.3 shells scripts which I know work for the Nvidia TX2 running JetPack 3.3. We've since then updated the OS for other drivers sets, but when I try to run it on JetPack 4.2 it seems to restart the device when I try running the standard ./openpose.bin image_dir /examples/media it will shutdown the TX2.

I personally think its an aarach issue as the caffe and openpose build scripts for JetPack 3.3 have the aarch for CUDA as 6.1, but I know that on previous builds I've created for OpenCV I've put this value as 6.2 specifically. I've tried this on the scripts but I still get the same result. It could be that we need a newer version of caffe built on the now aarch 6.2 as well as a new shell script built for aarch 6.2 for JetPack 4.2 builds which are running Ubuntu 18.04 L4T from Nvidia. 

The Device is the Nvidia Jetson TX2 same as before. 
I know this helps facilitate an understanding of our issues as the whole of JetPack has migrated to Ubuntu 18.04 the aarch is definitely different. One this to keep in mind as well if your looking to support the Jetson Xavier as well you the aarch for that is 7.2.

Cheers.
